### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`version++`
`polymer++`

### CODE REVIEW

**Comment by cdata** [comment]
Remove `enableCustomStyleProperties` at https://github.com/PolymerElements/paper-toggle-button/pull/5/files#diff-1cef05d826161c956d4fc799d670fff1R70

**Comment by cdata** [comment]
Remove first argument `this` at https://github.com/PolymerElements/paper-toggle-button/pull/5/files#diff-1cef05d826161c956d4fc799d670fff1L161

**Comment by notwaldorf** [comment]
Done the `customProperties` bit, I don't see the `this` you're talking about in the diff? :(

**Comment by notwaldorf** [comment]
ok, done did that one too

File: paper-toggle-button.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
 
 <dom-module id="paper-toggle-button">
 
-  <style is="x-style">
+  <style is="custom-style">
     * {
       --paper-toggle-button-unchecked-bar-color: #000000;
       --paper-toggle-button-unchecked-button-color: #f1f1f1;
```
Comment (cdata) [inline paper-toggle-button.html:44]: Inline defaults party :confetti_ball:

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**